### ISSUE TITLE
pana 0.20.0 is broken: 

### ISSUE THREAD

**Issue by kentcb** [dart-lang/pana#946]
Here is an excerpt from my CI logs:

```
2021-08-19T08:43:14.6305989Z Activated pana 0.20.0.
2021-08-19T08:43:14.8605309Z INFO       Running `/opt/hostedtoolcache/dart-sdk/bin/dart --version`...
2021-08-19T08:43:14.8937056Z INFO       Running `flutter --no-version-check --version --machine`...
2021-08-19T08:43:14.8982664Z WARNING    Unable to detect Flutter version.
2021-08-19T08:43:14.8983591Z            ProcessException: No such file or directory
2021-08-19T08:43:14.8984845Z              Command: flutter --no-version-check --version --machine
2021-08-19T08:43:14.9039434Z INFO       Running `git rev-parse --show-toplevel`...
2021-08-19T08:43:14.9144505Z SHOUT      Problem analyzing 
2021-08-19T08:43:14.9145908Z            FileSystemException: Directory listing failed, path = '/home/runner/work/humanizer/humanizer
2021-08-19T08:43:14.9147150Z            /' (OS Error: No such file or directory, errno = 2)
```

Pinning to `0.19.1` works as expected. It almost looks like the directory path is split over two lines...?

**Comment by isoos** [comment]
Yeah, that is a known regression, will be fixed in #945.

**Comment by isoos** [comment]
FYI: 0.20.1 has been released.
